stairway/src/test/java/bio/terra/stairway/fixtures/TestStairwayBuilder.java

```diff
@@ -138,6 +145,11 @@ public Stairway build() throws Exception {
       }
     }
 
+    // If we are treating existing stairways as running, then don't pass the recorded stairways
+    // into recoverAndStart for recovery.
+    if (existingStairwaysAreAlive) {
+      recordedStairways = Collections.emptyList();
```

Comment:
In JDK 9+ I'm now using `List.of()` instead of `Collections.emptyList()` (there's also `Map.of()`, `Set.of()`, etc)